### PR TITLE
Support for TSS files (Appcelerator Titanium)

### PR DESCRIPTION
**Pull request by m1ga**
Since the tss style files are "almost" css I use the same icons :smile: 

Screenshot_20210313_185839

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 [Unpublished]: https://github.com/file-icons/atom/compare/v2.1.47...HEAD
 
+[v2.1.48] - 2021-03-13
```
Comment (Alhadis) [inline CHANGELOG.md:8]:
```suggestion
[Unpublished]
```

The timestamped version-string headers are only for releases that've been published.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "file-icons",
 	"main": "lib/main.js",
-	"version": "2.1.47",
+	"version": "2.1.48",
```
Comment (Alhadis) [inline package.json:4]: Please revert that.

**Review by Alhadis — CHANGES_REQUESTED** [review]
Could you provide a bit of background on what TSS files are, and where/how they're used?

File: CHANGELOG.md (hunk 1 above)
Comment (m1ga) [reply CHANGELOG.md]: done

File: package.json (hunk 2 above)
Comment (m1ga) [reply package.json]: done

**Comment by m1ga** [comment]
Thank you for the quick feedback! Adjusted all your remarks, thanks!

TSS files are the style files inside the mobile app framework Appcelerator Titanium. 
The syntax is very similar to CSS but without the dashes between the words:

```
"TextField" : {
    borderWidth: 1,
    borderColor: "#000",
    left: 10,
    right: 10,
    borderRadius: 4,
    hintTextColor: "#888",
    fontSize: 14,
    color: "text",
    backgroundColor: "textfield"
}

"TextArea" : {
    borderWidth: 1,
    borderColor: "#000",
    left: 10,
    right: 10,
    borderRadius: 4,
    hintTextColor: "#888",
    fontSize: 14,
    color: "text",
    backgroundColor: "textfield"
}
```

Titanium uses XML for layouts, JS for code and TSS for styling elements inside the apps. All of the other files have nice icons just the styling TSS files had none. Assigning them to CSS makes sense since it is even inside a "style" folder and most dev's have a web dev background too.

Screenshot_20210313_192153

File: config.cson [hunk 3]
```diff
@@ -1751,6 +1751,7 @@ fileIcons:
 		match: [
 			[".css", "medium-blue", "css"]
 			[".less", "dark-blue", "less"]
+			[".tss", "medium-blue", "tss"]
```
Comment (Alhadis) [inline config.cson:1754]:
```suggestion
			[".tss", "dark-red", "tss"]
```

**Rationale:** [Appcelerator's logo](https://en.wikipedia.org/wiki/File:Appcelerator_logo.svg) uses red, and `.tss` files should ideally have a unique colour to distinguish them from CSS and Less files.

**Review by Alhadis — CHANGES_REQUESTED** [review]

File: config.cson (hunk 3 above)
Comment (m1ga) [reply config.cson]: I like that!

**Review by Alhadis — APPROVED** [review]

**Pull request merged**